syncd/NotificationProcessor.cpp

```diff
@@ -883,7 +883,18 @@ void NotificationProcessor::handle_tam_tel_type_config_change(
 
     SWSS_LOG_DEBUG("TAM telemesai_serialize_object_id(tam_type_id)try type config change on TAM id %s", data.c_str());
 
-    sendNotification(SAI_SWITCH_NOTIFICATION_NAME_TAM_TEL_TYPE_CONFIG_CHANGE, data);
+    sai_object_id_t rid;
+    sai_object_id_t vid;
+    sai_deserialize_object_id(data, rid);
+
+    if (false == m_translator->tryTranslateRidToVid(rid, vid))
```

Comment:
we dont use this kind of compare, move false to left or reverse condidtion

Reply:
Sure, FIxed.

I just copied an existing line.
![image](https://github.com/user-attachments/assets/c5e3e2ee-3ac3-4792-8dc9-c39cdab1cc0d)

Reply:
that was not my line :D i had to miss that from review